packages/web/src/pages/pay-and-earn-page/components/SalesTab.tsx

```diff
@@ -176,13 +184,30 @@ export const SalesTab = ({
 }: Omit<ReturnType<typeof useSales>, 'downloadCSV'>) => {
   const isMobile = useIsMobile()
   const mainContentRef = useMainContentRef()
+  const { color } = useTheme()
+  const { isEnabled: isNetworkCutEnabled } = useFeatureFlag(
+    FeatureFlags.NETWORK_CUT_ENABLED
+  )
 
   const columns = isMobile
     ? (['contentName', 'date', 'value'] as SalesTableColumn[])
     : undefined
 
   return (
     <div className={styles.container}>
+      {isNetworkCutEnabled ? (
+        <Flex gap='s' ph='l' pt='xl' alignItems='center'>
+          <IconMoneyBracket width={16} height={16} fill={color.neutral.n800} />
+          <Text variant='body' size='s' textAlign='left'>
+            {messages.networkSplitExplainer + ' '}
+            <ExternalTextLink to='' variant='visible'>
```

Comment:
You can apply all the text styles to external text link and drop the inner text! In fact the wrappping text sets body and size so you don't need to reset those

Reply:
That's what I was expecting but it didn't work 😕

Reply:
oh good to know, that sounds like a bug, will add a ticket